src/controllers/feedController.js

```diff
@@ -0,0 +1,71 @@
+const { feedService } = require("../services");
+const { keyCheck } = require("../utils/keyCheck");
+
+const getFeed = async(req, res) => {
+    const user = req.headers.Authorization;
+    const { page = 1, limit =10 } = req.query;
+    const offset = (page -1) * limit
+    const feedData = await feedService.getFeed(limit, offset, page);
+
+    return res.status(200).json({
+        message : "READ_SUCCESS",
+        data : feedData,
+    });
+};
+
+const addFeed = async(req, res) => {
+    const { content, challenge, imageUrl } = req.body;
+    const user = req.user.id;
```

Comment:
```suggestion
    const user = req.user.id;
    const { content, challenge, imageUrl } = req.body;
```
유저 정보를 먼저 꺼내오는 것이 보다 일반적인 순서입니다 :)